.github/workflows/slack-notify.yaml

```diff
@@ -0,0 +1,17 @@
+# This re-usable workflow file will trigger a Slack notification on the channel configure via the Webhook URL.
+# It currently comments a number of fields, but you can add or remove them as you see fit.
+#
+# To configure a Slack Webhook URL, follow the instructions here: https://api.slack.com/messaging/webhooks
+on:
+  workflow_call:
+
+jobs:
+  slack-notify:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: 8398a7/action-slack@v3
+        with:
+          status: 'failure'
```

Comment:
based on the docs maybe `status: ${{ job.status }}` is a better value here? Either that or maybe allow it as an input to be a bit more flexible. WDYT?

Reply:
I guess you'd need a specific one of the jobs to reference. I'm mainly hardcoding it since we only wanna send a notification for failures.

Reply:
happy enough to start out with that and we can worry about it if we need to in the future!